flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/StandaloneAutoscalerEntrypoint.java

```diff
@@ -94,4 +100,15 @@ JobAutoScaler<KEY, Context> createJobAutoscaler(
                 new RescaleApiScalingRealizer<>(eventHandler),
                 stateStore);
     }
+
+    @VisibleForTesting
+    static Configuration loadConfiguration(Optional<String> confOverrideDir, String[] args) {
+        if (confOverrideDir.isPresent()) {
+            return GlobalConfiguration.loadConfiguration(
+                    confOverrideDir.get(), ParameterTool.fromArgs(args).getConfiguration());
+        } else {
+            return GlobalConfiguration.loadConfiguration(
+                    ParameterTool.fromArgs(args).getConfiguration());
+        }
```

Comment:
Why do we need `CONF_OVERRIDE_DIR`?  As I understand, `FLINK_CONF_DIR` is enough.

Reply:
> Why do we need `CONF_OVERRIDE_DIR`? As I understand, `FLINK_CONF_DIR` is enough.

https://github.com/apache/flink-kubernetes-operator/blob/cb90b10c93241f9994acdd11d59ce1ec25ccbc47/flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/EnvUtils.java#L54C32-L54C53

I referred here

Reply:
> > Why do we need `CONF_OVERRIDE_DIR`? As I understand, `FLINK_CONF_DIR` is enough.
> 
> https://github.com/apache/flink-kubernetes-operator/blob/cb90b10c93241f9994acdd11d59ce1ec25ccbc47/flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/EnvUtils.java#L54C32-L54C53
> 
> I referred here

I thought carefully that adding an environment variable is indeed a bit redundant, because FLINK_CONF_DIR itself does not have a default configuration. for the opinion that FLINK_CONF_DIR is enough.